tui-markdown/src/lib.rs

```diff
@@ -224,14 +253,15 @@ where
         if self.needs_newline {
             self.push_line(Line::default());
         }
-        let style = match level {
-            HeadingLevel::H1 => styles::H1,
-            HeadingLevel::H2 => styles::H2,
-            HeadingLevel::H3 => styles::H3,
-            HeadingLevel::H4 => styles::H4,
-            HeadingLevel::H5 => styles::H5,
-            HeadingLevel::H6 => styles::H6,
+        let numeric_level: u8 = match level {
+            HeadingLevel::H1 => 1,
+            HeadingLevel::H2 => 2,
+            HeadingLevel::H3 => 3,
+            HeadingLevel::H4 => 4,
+            HeadingLevel::H5 => 5,
+            HeadingLevel::H6 => 6,
         };
+        let style = self.styles.heading(numeric_level);
```

Comment:
The enum's variants have numeric discriminants that are fine to use (https://docs.rs/pulldown-cmark/latest/pulldown_cmark/enum.HeadingLevel.html)
```suggestion
        let style = self.styles.heading(level as u8);
```